modules/apps/frontend-editor/frontend-editor-ckeditor-web/src/main/resources/META-INF/resources/editor/BalloonEditor.js

```diff
@@ -42,6 +42,15 @@ const BalloonEditor = ({config = {}, contents, name, ...otherProps}) => {
 		title: false,
 	};
 
+	const imageDirectionShim = editorConfig.toolbarImage.includes(
+		'ImageAlign'
+	) && {
+		style: {
+			float: 'left',
+			width: '100%',
```

Comment:
In case if the editor is empty with just one image

Reply:
With the change to `overflow: auto`, https://github.com/liferay-frontend/liferay-portal/pull/1339#discussion_r686061111,  this is not needed.

Reply:
https://user-images.githubusercontent.com/7663513/128912226-e047a5e0-b83b-457b-8163-f93a5fa93073.mov

nice! :D

Reply:
See https://github.com/liferay-frontend/liferay-portal/pull/1339/commits/df033efedceb335c9f7712859befaa2bb5003145